stable/kong/README.md

```diff
@@ -81,6 +81,7 @@ and their default values.
 | proxy.type                     | k8s service type. Options: NodePort, ClusterIP, LoadBalancer                     | `NodePort`          |
 | proxy.loadBalancerSourceRanges | Limit proxy access to CIDRs if set and service type is `LoadBalancer`            | `[]`                |
 | proxy.loadBalancerIP           | To reuse an existing ingress static IP for the admin service                     |                     |
+| proxy.externalIPs              | List of IP addresses                                                             | `[]`                |
```

Comment:
Can you update the doc line with some more detail to help our users out?

Reply:
Just added more detail to the doc line. How about that?